markets/bfp-market/contracts/modules/OrderModule.sol

```diff
@@ -193,50 +198,55 @@ contract OrderModule is IOrderModule {
         );
 
         PerpMarket.Data storage market = PerpMarket.exists(marketId);
+        Runtime_commitOrder memory runtime;
 
         if (market.orders[accountId].sizeDelta != 0) {
             revert ErrorUtil.OrderFound();
         }
 
         validateOrderHooks(hooks);
 
-        uint256 oraclePrice = market.getOraclePrice();
+        runtime.oraclePrice = market.getOraclePrice();
 
         PerpMarketConfiguration.Data storage marketConfig = PerpMarketConfiguration.load(marketId);
 
+        // NOTE: `oraclePrice` in TradeParams should be `pythPrice` as to track the raw Pyth price on settlement. However,
+        // we are only committing the order and the `trade.newPosition` is discarded so it does not matter here.
+        runtime.params = Position.TradeParams(
```

Comment:
Pedantic: Can we rename this to `tradeParams` instead of just `params` (same in settlement)